### PR TITLE
[skip-ci] Update fonts' documentation (CMS request)

### PR DESCRIPTION
**Pull request by couet**
Better description of fonts' names.

### CODE REVIEW

File: core/base/src/TAttText.cxx [hunk 1]
```diff
@@ -229,21 +229,21 @@ printout given by this command:
...
-       6 :       helvetica-bold-r-normal    "Arial"
-       7 :       helvetica-bold-o-normal    "Arial"
-       8 :       courier-medium-r-normal    "Courier New"
-       9 :       courier-medium-o-normal    "Courier New"
-      10 :       courier-bold-r-normal      "Courier New"
-      11 :       courier-bold-o-normal      "Courier New"
-      12 :       symbol-medium-r-normal     "Symbol"
-      13 :       times-medium-r-normal      "Times New Roman"
-      14 :                                  "Wingdings"
+   Font number         X11 Names             TTF Names                  PDF Names
```
Comment (vepadulano) [inline core/base/src/TAttText.cxx:232]: The new names look good, but could we add some documentation or links to the original source for `X11` names or `TTF` or `PDF` names. From my perspective, reading `TTF Names` or the others doesn't really tell me much about these names. Who decided what `TTF Names` are? Is there some documentation somewhere in the original protocols/repositories? Similarly for X11 and PDF

**Review by vepadulano — CHANGES_REQUESTED** [review]
The update looks good, thanks! Can we try to make it even more clear for external users by pointing to where these nomenclatures were originally defined?

**Comment by couet** [comment]
I can add the explanations I put here: https://github.com/root-project/root/issues/14064

**Comment by couet** [comment]
I have revisited this part of the doc.

**Review by vepadulano — APPROVED** [review]
Thanks Olivier, looks good to me now!

**Pull request merged**